lnst/Recipes/ENRT/ConfigMixins/DisableIdleStatesMixin.py

```diff
@@ -52,6 +53,7 @@ def generate_sub_configuration_description(self, config):
 
     def remove_sub_configuration(self, config):
         for host in self.disable_idlestates_host_list:
-            host.run("cpupower idle-set -E")
+            if self.params.minimal_idlestates_latency >= 0:
```

Comment:
I'd change this to: `if getattr(self.params, "minimal_idlestates_latency", None)`